lib/widgets/album_photos_list.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import '../models/photo_model.dart';
+import '../theming/app_dimensions.dart';
+import '../widgets/photo_tile.dart';
+import '../widgets/universal_app_divider.dart';
+
+class AlbumPhotosList extends StatelessWidget {
+  final List<PhotoModel> photos;
+
+  AlbumPhotosList({required this.photos});
+
+  @override
+  Widget build(BuildContext context) {
+    return Flexible(
```

Comment:
@AndreiBorzaRodeapps a Flexible can we used only in Row or Column. What do you think will happen if someone will put this in another widget?